modules/builds-secrets-restrictions.adoc

```diff
@@ -10,7 +10,12 @@ To use a secret, a pod needs to reference the secret. A secret can be used with
 * As files in a volume mounted on one or more of its containers.
 * By kubelet when pulling images for the pod.
 
-Volume type secrets write data into the container as a file using the volume mechanism. `imagePullSecrets` use service accounts for the automatic injection of the secret into all pods in a namespaces.
+Volume type secrets write data into the container as a file using the volume mechanism. `imagePullSecrets` use service accounts for the automatic injection of the secret into all pods in a namespaces. 
```

Comment:
Hi Steven, seem we got comments in https://github.com/openshift/openshift-docs/pull/83900/files#r1810886633 
`s/namespaces/namespace`